webcompat/static/js/lib/models/label-list.js

```diff
@@ -22,7 +25,7 @@ var issues = issues || {}; // eslint-disable-line no-use-before-define
  * new issues.LabelList({url:'/path/to/labels.json'});
  */
 
-issues.LabelList = Backbone.Model.extend({
+const LabelList = Backbone.Model.extend({
```

Comment:
<s>Question: in issue.js, we do `export const Issue = Backbone.Model.extend` and `export const IssueCollection = Backbone.Collection.extend`. 

So why don't we `export` `LabelList`?</s>

I answered by scrolling down. 😄 

We export it later: `export { LabelList };`